### ISSUE TITLE
Will there come any new tutorials?

### ISSUE THREAD

**Issue by MichaelBrauner** [gregurco/jobeet-tutorial#31]
I look almost every day onto the Jobeet page for month now. 
I am wondering if the rest of the tutorials will be uploaded. 

Best regards
Michael

**Comment by gregurco** [comment]
Hello @Slowwie 
Of course I plan to publish the rest of tutorial and partially it's done for now. But at the moment I'm a little bit busy and can't finish it in coming days.
Your interest motivates me to find the time as soon as possible.
Thank you!

**Comment by Arielblues** [comment]
Hi i am also interested in the rest off the tutorial.  
Keep up the good work

**Comment by MichaelBrauner** [comment]
Yes, I am very interested in this tutorial. I am a Symfony programmer since 5 years. But I really learned new things - the SendGrid thing is very nice. I always had problems with my mails on certain providers (hotmail). Now that's history. Also I tried that docker thing. I thing its amazing. I struggle now to deploy that to AWS Fargate but I will do also thing in a while. All this I learned from this tutorial. So I am very interested in the last parts ;). It´s amazing.

**Comment by lototskyi** [comment]
Hi, thank you for these useful tutorials. I started learning symfony few months ago and they were very helpful for me. I'm looking forward to the rest of the tutorials.

**Comment by gregurco** [comment]
Upgraded the tutorial to Symfony 4.2 in PR #32

**Comment by onlinebizsoft** [comment]
Looking forward for the rest of tutorial :)

**Comment by lex111** [comment]
I'm anticipate with pleasure new updates! :heart_eyes:

**Comment by maxsmooth** [comment]
Hi from Italy. I am new to Symfony. I am also interested in the rest off the tutorial.
Sorry for bad english.

**Comment by gregurco** [comment]
Latest update:
@sergey-bleih helps with tests part. So you can see PR with code for unit tests: https://github.com/gregurco/jobeet/pull/8

**Comment by Evgeny1973** [comment]
Пора уже на 4.3 обновляться ;)

**Comment by MichaelBrauner** [comment]
I am very interested in the deployment part. I wonder, how u deploy such docker containers. 
I watch out for updates of this series now since November last year. I guess there is not really hope this tutorial will be completed, isn't it?

**Comment by gregurco** [comment]
@Slowwie thank you for question and interest. I will continue tutorial as soon as will have enough free time.
For deployment part: it's quite hard part, especially with docker, but you can read more about AWS / Azure / Google Cloud Engine - all they have integration for containers (docker).

**Comment by MichaelBrauner** [comment]
Yes, I know. I managed to deploy my project with AWS Fargat (EC2 would be too much maintenance work). But its quite expensive. I was wondering if u can present some cheaper solution in your tutorial.

**Comment by gregurco** [comment]
@Slowwie I don't think that docker is right solution for cheap target. It is good, stable, secure, adjusted for complex architectures, BUT it's not cheat. In case you would like to deploy production with docker and not in cloud (aws/azure/gce) then you will have to study a bunch of tools/technologies besides docker and to rent not just 1 server (at least one dedicated for orchestrator and one for registry). So docker in production probably is not proper solution for small projects (IMHO).

**Comment by Saint-Cyr** [comment]
@gregurco Thank you so much.
What about the search engine feature ? do you plan anything for it ? I think it could be amazing to use [algolia](https://www.algolia.com/) or Elastica.

**Comment by rodrigobfournier** [comment]
@gregurco very good initiative to rewrite this tutorial in a more modern framework version, but, is there any possibility to you create this tutorial with SF 5?
